### PR TITLE
BTA-6762 Rebrand API documentation

### PR DESCRIPTION
**Pull request by sayidu**
* Updates text instances of the TransferZero name with AZAFinance.

Screenshot 2021-07-02 at 16 44 33
Screenshot 2021-07-02 at 16 49 16

### CODE REVIEW

File: _docs/architecture.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Generally in our API there are two types of status codes used:
 
 # Webhooks
 
-A *webhook* is a callback which enables our API to notify your system of new events concerning your transactions. Whenever objects in the TransferZero API change state, we can optionally send the changed details to a pre-registered webhook address. Webhooks can be created either using the [developer portal](https://developers.transferzero.com), or using the [TransferZero API](https://api.transferzero.com/documentation#webhooks).
```
Comment (archmagos) [inline _docs/architecture.md:22]: Should the second link read AZA Finance API?

File: index.html [hunk 2]
```diff
@@ -13,8 +13,8 @@ <h1>{{ site.title }}</h1>
 <div class="container">
     <div class="row">
       <div class="col-md-6">
-        <h2 class="header-light regular-pad">About TransferZero</h2>
-          <p><a href="https://transferzero.com/">TransferZero</a> is a comprehensive API for hybrid financial infrastructure across Africa and Europe, offering both foreign exchange and last mile distribution.</p>
+        <h2 class="header-light regular-pad">About AZA Finance</h2>
```
Comment (archmagos) [inline index.html:16]:
```suggestion
        <h2 class="header-light regular-pad">About AZA Finance API</h2>
```

File: index.html [hunk 3]
```diff
@@ -13,8 +13,8 @@ <h1>{{ site.title }}</h1>
...
+          <p><a href="https://transferzero.com/">AZA Finance</a> is a comprehensive API for hybrid financial infrastructure across Africa and Europe, offering both foreign exchange and last mile distribution.</p>
```
Comment (archmagos) [inline index.html:17]:
```suggestion
          <p><a href="https://transferzero.com/">AZA Finance</a> offer a comprehensive API for hybrid financial infrastructure across Africa and Europe, offering both foreign exchange and last mile distribution.</p>
```

**Review by archmagos — COMMENTED** [review]
The designs seem unclear on whether the main page title should read TransferZero or AZA Finance API. It makes logical sense that it would be the latter, as you've done it. Happy to approve, just a couple of comments left.

**Review by archmagos — APPROVED** [review]

**Review by cjjharries — APPROVED** [review]
Everything looks as intended, approved from the PD and marketing sides.

**Pull request merged**